core/project.cpp

```diff
@@ -25,6 +25,22 @@ QString LocalProject::fullName() const
   return dir.dirName();
 }
 
+bool LocalProject::hasConflictedCopies()
+{
+  if ( isValid() )
+  {
+    int count = 0;
+    QDirIterator it( projectDir, QStringList() << "*conflicted copy*", QDir::Files | QDir::NoDotAndDotDot, QDirIterator::Subdirectories );
+    while ( it.hasNext() )
+    {
+      it.next();
+      count++;
+    }
+    return count > 0 ? true : false;
+  }
+  return false;
+}
```

Comment:
shall we ignore .mergin directory, just in case?

Reply:
At least on unix-based systems `QDirIterator` by default ignores hidden files and directories. But to be safe additional check was added.